Lib/heapq.py

```diff
@@ -599,9 +599,3 @@ def nlargest(n, iterable, key=None):
     from _heapq import _heappop_max
 except ImportError:
     pass
-
-
-if __name__ == "__main__":
```

Comment:
I saw that `pragma: nocover` gets used elsewhere (e.g. https://github.com/python/cpython/blob/5affd23e6f42125998724787025080a24839266e/Lib/uuid.py#L495) so I've done that.